fastlane/Fastfile

```diff
@@ -73,11 +73,13 @@ platform :android do
     version = android_get_version_name(gradle_file: gradle_file_path)
     puts "Deploying #{version}"
     gradle(
-      tasks: ["androidSourcesJar", "androidJavadocsJar", "uploadArchives"],
+      tasks: [
+          "androidSourcesJar", "androidJavadocsJar", "uploadArchives"
+      ],
       properties: {
-        "signing.keyId" => ENV['GPG_SIGNING_KEY_ID'],
-        "signing.password" => ENV['GPG_SIGNING_KEY_PW'],
-        "signing.secretKeyRingFile" => "./secring.gpg",
+        "signing.keyId" => ENV['GPG_SIGNING_KEY_ID_NEW'],
+        "signing.password" => ENV['GPG_SIGNING_KEY_PW_NEW'],
```

Comment:
I have created a new key and added to circleci. I haven't removed the previous one just in case we need it (we shouldn't). Should we revoke the previous secret key that's also being removed in this PR? I am worried people have issues when installing a previous version if the key used to sign that version is revoked.